src/gui/book/Book.cpp

```diff
@@ -1448,7 +1454,7 @@ void ARX_INTERFACE_ManageOpenedBook() {
 				break;
 			}
 			case BOOKMODE_QUESTS: {
-				gui::manageQuestBook();
+				g_playerBook.journal.render();
```

Comment:
The game text (tutorial, key binding, etc.) refers to this as the quest book so maybe we should keep the name. Not entirely sure though.